index.js

```diff
@@ -0,0 +1,97 @@
+
+document.addEventListener('DOMContentLoaded', function() {
+    // JavaScript code
+    const today = new Date();
+    const thisYear = today.getFullYear();
+    const footer = document.querySelector('footer');
+    const copyright = document.createElement('p');
+    copyright.innerHTML = `&#169; Viktorija Kurmisa ${thisYear}`;
+    footer.appendChild(copyright);
+});
+
+/////////////////////////////////////////
```

Comment:
This section is very neat and well organized; however, there are too many comments. Comments really should be used to convey additional information about the code - explaining coding/design decisions.